src/components/AppProvider/AppProvider.tsx

```diff
@@ -107,6 +107,7 @@ export class AppProvider extends React.Component<AppProviderProps, State> {
       children,
     } = this.props;
     const {intl, appBridge, link} = this.state;
+    const {colorScheme, ...restOfTheme} = theme;
```

Comment:
It seems odd that the theme we pass into the AppProvider differs from the theme we pass into the ThemeProvider.

I think this would be clearer if AppProvider's theme prop was equal to either: The theme prop of ThemeProvider, or the whole props object of ThemeProvider. Then in the future if we added new items we wouldn't need to adjust this bit of passthrough logic. You'll also avoid needing to muck about with new types like `ThemeProviderThemeConfig`.

```jsx
<ThemeProvider theme={theme}>....</ThemeProvider>

// or

<ThemeProvider ...theme>....</ThemeProvider>
```

The second option is teeechnically a breaking change but I don't think we ever supported people using ThemeProvider standalone externally anyway.

Reply:
While the API is a bit more ergonomic when you just want to set the `colorScheme`, I don't feel the increase in complexity is worth it to pull our `colorScheme` into its own prop on the `ThemeProvider`.

How strongly do you feel this should be done, @tmlayton?

Reply:
I'm okay with that